09_variables_conditionals/solution_student/temp_converter/js/main.js

```diff
@@ -0,0 +1,38 @@
+$('#fahrenheit_to_celsius').on('click', function() {
+	var inputTemp = $('#temperature').val();
+	inputTemp = parseFloat(inputTemp);
+	var cTemp = inputTemp - 32 / 1.8; 
+	$('#result').html(cTemp + ' °C ');
+
+
+		if (cTemp > 10) {
+			$('body').addClass('hot');
+			$('body').removeClass('cold');
+	} else if (cTemp < 10) {
+			$('body').addClass('cold');
+			$('body').removeClass('hot');
+	// I couldn't figure this part out. Was I on the right track? 
```

Comment:
You were very close! The only reason this isn't working is because you are using `parseFloat' on the `inputTemp`, so it's converting the string to a number, which you absolutely should be doing, so good on you for that, but in this case if the input is empty it is going to return a value of `NaN` (not a number), instead of an empty string. So, to check if it's NaN, which conveniently works if the value is words instead of numbers as well, we can do `isNaN(inputTemp)' — which will return `true` if the value is not a number (like an empty input). But you had the logic down right!